Add VMWare Tanzu notes. Also differentiate between supported and tested k8s (#719)
stackabletech/documentation: modules/ROOT/pages/kubernetes/microk8s.adoc

= Microk8s

Microk8s uses a non-standard Kubelet state directory.

> Therefore installing secret-operator and listener-operator on Microk8s requires a special treatment.
> Please read on the xref:secret-operator:installation.adoc#_microk8s[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_microk8s[listener-operator installation guide] for details.

Reviewer: ```suggestion
Therefore, installing the secret-operator and listener-operator on Microk8s requires special handling.
Please refer to the xref:secret-operator:installation.adoc#_microk8s[secret-operator installation guide] and the xref:listener-operator:installation.adoc#_microk8s[listener-operator installation guide] for details.
```
